data/learnsets.js

```diff
@@ -64993,6 +64993,57 @@ let BattleLearnsets = {
 		wish: ["7L22"],
 		worryseed: ["7T"],
 	}},
+	pokestarsmeargle: {learnset: {
+
```

Comment:
Style has never been to add empty lines here. If we were to do these entries, we should do them as one-liners:

```
pokestarsmeargle: {learnset: {}},
```

Reply:
But this is irrelevant. I would rather autofill these in the `Data.Template` constructor, than add empty entries here.

Reply:
Alright, sounds good.